zlog_plugin.go

```diff
@@ -1,89 +0,0 @@
-package dmetering
-
-import (
```

Comment:
Don't we want to keep this one, it was quite useful for testing purposes. I'm not against the delete, but it seems we could easily keep it.

Reply:
this functionality is now in the `loggerEmitter`

Reply:
I see, I missed it indeed. I preferred to have them in their own file, it make it easily visible from the project structure what are the valid implementation of an Emitter. But you moved them, so maybe you preferred having all them in a file. I leave you the decision here.